### ISSUE TITLE
feature request: Windows Hello support

### ISSUE THREAD

**Issue by stevelitchfield** [twsouthwick/KeePassWin#59]
This would be really useful, e.g. On phones, where it's harder to put in a long master password.

**Comment by GitMoDu** [comment]
I agree, this must-have feature. I'm dabbling on some experiments to implement it, but I am unfamiliar with this internal-shell approach of the code.

**Comment by twsouthwick** [comment]
I like this idea and have it on the back log (see #4), I just haven't had time to do it myself. The shell approach I use is from the [Prism project](https://github.com/PrismLibrary/Prism). I would suggest looking into the `DialogCrednetialProvider` which implements `ICredentialProvider`. This would make it easy to inject to where it needs.

**Comment by GitMoDu** [comment]
Did some quick and dirty tests, my local build of KeePassWin remembers my password on runtime only and asks for Hello Authentication to use said stored password, with no extra prompts (** cough ** PassKeep **cough***).

I'll look into Prism, but I'd really like to know is WHY you chose to use Prism on this project, instead of keeping everything neat and native-like (this is a security open-source project, after all).

**Comment by GitMoDu** [comment]
Ok, I did a bit of digging and came up with this: https://github.com/twsouthwick/KeePassWin/pull/61

Let me know your thoughts.
